pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1748,4 +1738,26 @@ private void assertInvalidRetentionPolicyAsPartOfAllPolicies(Policies policies,
             assertTrue(e.getMessage().startsWith("Invalid retention policy"));
         }
     }
+
+    @Test
+    public void testNamespaceResourceGroup() {
+        String namespace = BrokerTestUtil.newUniqueName(this.testTenant + "/namespace");
+        ResourceGroup testResourceGroup = new ResourceGroup();
+        testResourceGroup.setDispatchRateInBytes(10000L);
+        testResourceGroup.setDispatchRateInMsgs(100);
+        testResourceGroup.setPublishRateInMsgs(100);
+        testResourceGroup.setPublishRateInBytes(10000L);
+        final String resourceGroupName = "test-resource-group";
+        try {
```

Comment:
Remove `try...catch`

Reply:
This change will cause a compile error.

Reply:
A easy way: `public void testExistNamespaceResourceGroup() throws Exception {`